appeals/web/src/server/appeals/appeal-details/address/address.controller.js

```diff
@@ -0,0 +1,78 @@
+import logger from '#lib/logger.js';
+import { addNotificationBannerToSession } from '#lib/session-utilities.js';
+import { changeSiteAddressPage } from './address.mapper.js';
+import { changeSiteAddress } from './address.service.js';
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+export const getChangeSiteAddress = async (request, response) => {
+	return renderChangeSiteAddress(request, response);
+};
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+const renderChangeSiteAddress = async (request, response) => {
+	const { errors } = request;
+
+	const backLinkUrl = request.originalUrl.split('/').slice(0, -3).join('/');
+
+	const appealDetails = request.currentAppeal;
```

Comment:
Again just a suggestion, you could destructure this along with the `errors` variable above, to avoid creating this variable (or just feed `request.currentAppeal` into the mapper function directly)